### PR TITLE
DHFPROD-4733: Get Record from database

### PR DESCRIPTION
**Pull request by rahulvudutala**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/getRecordTest.sjs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  ];
+}
+
+function getNonExistentRecordTest() {
+  const docUri = "/content/nonExistentRecord.json";
+  const record = entitySearchService.getRecord(docUri);
+  console.log(record);
+}
+
+[]
```
Comment (ryanjdew) [inline marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/getRecordTest.sjs:91]: Should these be uncommented now? It doesn't appear that anything is being tested here.

Comment (rahulvudutala) [reply marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/getRecordTest.sjs:93]: Thank you for noticing it. I updated it

**Review by ryanjdew — APPROVED** [review]

**Review by Sanjeevani19 — CHANGES_REQUESTED** [review]
Looks like some unit tests are failing, @rahulvudutala please fix them.

**Comment by rahulvudutala** [comment]
> Looks like some unit tests are failing, @rahulvudutala please fix them.

I just pushed some changes

**Review by Sanjeevani19 — APPROVED** [review]

**Review by ryanjdew — APPROVED** [review]

**Pull request merged**